FetchRequests/Sources/SwiftUI/FetchableRequest.swift

```diff
@@ -107,17 +110,29 @@ public struct FetchableRequest<FetchedObject: FetchableObject>: DynamicProperty
         let controller = fetchController
         let binding = $wrappedValue
         let animation = self.animation
-
+        let initialLoad = $initialLoad
         subscription.value = fetchController.objectDidChange.sink { [weak controller] in
             guard let controller else {
                 return
             }
             withAnimation(animation) {
```

Comment:
Should probably be checking `initialLoad.wrappedValue` before starting the animation block, no?
It's also super surprising that the first pass doesn't work.